.gitignore

```diff
@@ -173,3 +173,4 @@ docs/langchain.md
 docs/mistral.md
 docs/openai.md
 docs/examples/langchain/penguins.csv
+docs/index.md
```

Comment:
The `index.md` file is now generated by the script. I could find no other way to include the video both in the README and the docs.